app/views/work_orders/index.html.erb

```diff
@@ -0,0 +1,90 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Work Orders for 10/1/2019</h1>
+
+<div class="master">
+
+
+  <div>
+    <div><br></div>
+
+    <%  @minute.each do |hour| %>
+      <div class="flex_time" id="hour<%= hour %>"><%= hour %></div>
+
+
+    <%end %>
+  </div>
+
+
+  <% @technicians.each do |technician| %>
+    <section class="flex-items"  style="height: 1710px; position: relative" ><%= technician.name %>
+      <% @prior_work_time =  @work_orders[0].time%>
+      <% @prior_work_duration = 0 %>
+      <% @duration_height = 0 %>
+      <% @work_order_duration_height = 0 %>
+      <% @work_orders.each  do  |work_order|  %>
+        <% next unless work_order.technician_id == technician.id %>
+        <% first_work_time = work_order.time %>
+        <% empty_id = @prior_work_time - first_work_time%>
+        <% prior_height = @duration_height%>
+        <div class="empty_box"
+             onclick="showAlert(<%= ((((work_order.time.strftime('%H').to_i) + ((work_order.time.strftime('%M').to_i)*(1.0/100)*(7/3)))*140)-(840+prior_height))%>)" id = "<%= empty_id %>"
+             style="position: absolute;
+               width: 100%;
+               top:<%=((((@prior_work_time.time.strftime('%H').to_i)-6)*140)+(((@prior_work_time.time.strftime('%M').to_i)/60.0)*140)+18) + ((@prior_work_duration/60.0)*140)%>px;
+               height:<%= (((((work_order.time.strftime('%H').to_i)) + (((work_order.time.strftime('%M').to_i)/60.0)))*140)-(840+prior_height))%>px">
```

Comment:
This is not great. You should avoid using magic numbers as much as possible, but especially a dozen in 5 lines.